azext_iot/tests/central/__init__.py

```diff
@@ -382,6 +383,60 @@ def _delete_organization(self, api_version, org_id):
             ],
         )
 
+    def _create_destination(self, api_version, dest_id):
+        self.kwargs["authorization"] = json.dumps(
+            {
+                "type": "connectionString",
+                "connectionString": STORAGE_CSTRING,
+                "containerName": STORAGE_CONTAINER,
+            }
+        )
+        command = "iot central export destination create --app-id {} \
+            --dest-id {} --type {} --name '{}' --authorization '{}'".format(
+            APP_ID,
+            dest_id,
+            "blobstorage@v1",
+            "Blob Storage",
+            "{authorization}",
+        )
+        return self.cmd(
+            command, api_version=api_version, checks=[self.check("id", dest_id)]
+        ).get_output_in_json()
+
+    def _delete_destination(self, api_version, dest_id):
+        command = (
+            "iot central export destination delete --app-id {} --dest-id {}".format(
+                APP_ID, dest_id
+            )
+        )
+        self.cmd(command, api_version=api_version)
+
+    def _create_export(self, api_version, export_id):
+
+        self.kwargs["enrichments"] = json.dumps({"Simulated": {"path": "$simulated"}})
+        self.kwargs["destinations"] = json.dumps([{"id": "aztestdest0001"}])
```

Comment:
Should the destination Id value be parameterized `{"id": "aztestdest0001"}` ? Looking to reduce hardcoded strings where possible.

Reply:
fixed.